src/Game/Systems/ScoreScreenSystem.cpp

```diff
@@ -0,0 +1,161 @@
+#include "Game/Systems/ScoreScreenSystem.h"
+
+
+ScoreScreenSystem::ScoreScreenSystem(SystemParams params)
+    : System(params)
+    , PureSystem("ScoreScreen")
+{
+    EVENT_SUBSCRIBE_MEMBER(m_EPlayerDeath, &ScoreScreenSystem::OnPlayerDeath);
+    EVENT_SUBSCRIBE_MEMBER(m_EPlayerSpawned, &ScoreScreenSystem::OnPlayerSpawn);
+    EVENT_SUBSCRIBE_MEMBER(m_EPlayerConnected, &ScoreScreenSystem::OnPlayerConnected);
+    EVENT_SUBSCRIBE_MEMBER(m_EPlayerDisconnected, &ScoreScreenSystem::OnPlayerDisconnected);
+}
+
+void ScoreScreenSystem::UpdateComponent(EntityWrapper& entity, ComponentWrapper& scoreScreen, double dt)
+{
+    if (!IsServer) {
+        return;
+    }
+
+    if(!entity.HasComponent("ScoreScreen")){
+        return;
+    }
+
+    int redTeamEnum;
+    int blueTeamEnum;
```

Comment:
Are still not used:
redTeamEnum = (int)cTeam["Team"].Enum("Red");
blueTeamEnum = (int)cTeam["Team"].Enum("Blue");
spectatorTeamEnum = (int)cTeam["Team"].Enum("Spectator");